src/routes/index.tsx

```diff
@@ -10,6 +10,7 @@ export default component$(() => {
   return (
     <>
       <Hero />
+      <h1>Hello im here, its emma</h1>
```

Comment:
this is just not emma, it needs to be small living

Reply:
aaah my bad